zulipterminal/ui_tools/views.py

```diff
@@ -547,6 +547,18 @@ def get_next_unread_pm(self) -> Optional[int]:
             return pm
         return None
 
+    def update_msg_list_presence_markers(self) -> None:
+        for index in range(len(self.body.log)):
```

Comment:
Since you're using `self.body.log` more than once, you could extract it in a variable.

Reply:
Yes, though I think it would be not helpful for the line L558 (where we're updating `self.body.log` itself)?

Or do you mean updating the whole message list outside the loop in the end:
`self.body.log = msg_list` (if `msg_list` is the variable for instance) ?

Reply:
I see that you're only mutating the attribute inside the loop, so extracting `self.body.log` would work.

Reply:
(Assuming by helpful you meant it is not going to update the original log.)